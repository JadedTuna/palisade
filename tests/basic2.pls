in {}
out {
    low err: bool;
}

num := -3;
total := 34;
err = false;
try {
    if (num < 0) {
        throw;
    }
    total = total + num;
} catch {
    err = true;
}
