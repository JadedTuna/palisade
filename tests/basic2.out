[[1;32m OK [0m] [1;93mlow [0m [1;34merr[0m is [1;93mlow[0m
